<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="baudRateComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serialbox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>串口选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>算法类型</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>插补模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>象限</string>
    </property>
   </widget>
   <widget class="QComboBox" name="algorithmComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="interpTypeComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="quadrantComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>细分</string>
    </property>
   </widget>
   <widget class="QComboBox" name="subdivComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>起点坐标</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>终点坐标</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="x_s">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="y_s">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="x_f">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="y_f">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>450</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="run">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="commandoutputBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="speedbox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>速度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draw">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>绘制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>430</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>圆心坐标</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="x_c">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="y_c">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>500</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="commandoutput">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>141</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QChartView" name="chartView">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts/QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
